cli/src/commands/init.rs

```diff
@@ -8,7 +8,7 @@ use clap::parser::ValuesRef;
 use clap::ArgMatches;
 use dialoguer::{Confirm, FuzzySelect, Input, MultiSelect};
 use phylum_lockfile::LockfileFormat;
-use phylum_project::{LockfileConfig, PROJ_CONF_FILE};
+use phylum_project::{LockfileConfig, PROJ_CONF_FILE, ProjectConfig};
```

Comment:
```suggestion
use phylum_project::{LockfileConfig, ProjectConfig, PROJ_CONF_FILE};
```
CI is failing w/o this change